src/oca_github_bot/tasks/migration_issue_bot.py

```diff
@@ -58,6 +58,10 @@ def _set_lines_issue(gh_pr, issue, module):
                 lines.append(new_line)
                 added = True
         lines.append(line)
+
+    # make the addition working on an empty migration issue
+    if not added:
+        lines.append(new_line)
```

Comment:
A typical migration issue ends with comments after the check list of modules:

![image](https://user-images.githubusercontent.com/692075/176997431-26185e7f-bd43-4c5d-bd83-703084b7f1e4.png)

So this would add a line after the comment which may not look very nice.

I wish we had better tests for `_set_lines_issue`.

Reply:
> A typical migration issue ends with comments after the check list of modules:

that's not the point of the PR. 

The line will be added only if the issue doesn't contain module list.
In your screenshot, my patch has no effect, because previous code will add the line in the correct place, and so added will be true.

That is just a trivial patch to initialize migration issue, if there is no module list.